Source/Core/Common/StringUtil.cpp

```diff
@@ -247,6 +247,25 @@ bool TryParse(const std::string& str, u32* const output)
   return true;
 }
 
+bool TryParse(const std::string& str, u64* const output)
```

Comment:
I'm really surprised, is there nothing in the standard that would give us similar functionality?